base.html.j2

```diff
@@ -110,6 +110,7 @@
       <li><a href="{{ "sponsors.html" | link }}">Sponsors</a>
       <li><a href="{{ "sponsor-levels.html" | link }}">Sponsorship Opportunities</a>
       <li><a href="{{ "about.html" | link }}">About</a>
+      <li><a href="{{ "/coc" | link }}">Code of Conduct</a>
```

Comment:
Does this work, or do we need it to have the `.html` appended to it, e.g.
```suggestion
      <li><a href="{{ "/coc.html" | link }}">Code of Conduct</a>
```

Reply:
What i have does work, the /coc/index.html page is in a subdir.
I did this because the contract has the https://2025.pgconf.dev/coc URL not https://2025.pgconf.dev/coc.html URL and some of the contracts are already signed.